### PR TITLE
Minor Formatting changes to AppDynamics Config section

### PR DESCRIPTION
**Pull request by okilker**

### CODE REVIEW

File: src/docs/components/otlp-exporter.mdx [hunk 1]
```diff
@@ -51,7 +51,7 @@ and expects to receive a response with each request. You can learn more about th
 OpenTelemetry can export traces, logs, and metrics to various backends to analyze in order to understand an application’s performance 
 and behavior. There are multiple monitoring backends (also known as end-points) which can support OpenTelemetry using the OTLP protocol. 
 
-In this section, we will getting started configurations to the Collector to export telemetry data to **AppDynamics, Honeycomb, Lightstep, and SumoLogic** end-points.
+In this section, we share getting started configurations to the Collector to export telemetry data to **AppDynamics, Honeycomb, Lightstep, and SumoLogic** end-points.
```
Comment (anuraaga) [inline src/docs/components/otlp-exporter.mdx:54]:
```suggestion
In this section, we share basic configurations to the Collector to export telemetry data to **AppDynamics, Honeycomb, Lightstep, and SumoLogic** end-points.
```

Don't think `getting started` is a clear adjective

**Review by anuraaga — APPROVED** [review]
Thanks!

File: src/docs/components/otlp-exporter.mdx (hunk 1 above)
Comment (okilker) [reply src/docs/components/otlp-exporter.mdx:54]: Hi @anuraaga , agreed this content was pre-existing, and I didn't know what the original content meant to say. Please change to original intent if you have access to that content : )

**Comment by alolita** [comment]
@okilker thanks for making the changes.

**Review by alolita — APPROVED** [review]

**Pull request merged**